Calendars/Calendars.Plugin.Android/CalendarEventReminderExtensions.cs

```diff
@@ -2,11 +2,13 @@
 using Android.Provider;
 using Plugin.Calendars.Abstractions;
 
+#nullable enable
```

Comment:
I'm getting a lot of those if `#nullable enable` is removed

![image](https://user-images.githubusercontent.com/6609929/127054867-1a0ed8a2-ee8a-497e-b234-eb41f3160c84.png)

(everything still builds though)

Reply:
That is very curious. What version of VS are you using?

Perhaps VS4Mac just isn't as strict about that warning... though I thought for things like this they both used the same code thanks to Roslyn...

For _science_ I'd be somewhat curious if putting the config in those project files (rather than just relying on directory.build.props) would make a difference... but I s'pose `#nullable enable` isn't the end of the world...

Reply:
I'm using latest VS 2019 Enterprise (16.10.4). Interesting thing is that I don't have this probe in my Xamarin project that targets `.netstandart2.1`.

Reply:
Alright, thanks for checking.